numpy/polynomial/_polybase.py

```diff
@@ -280,25 +280,25 @@ class as self with identical domain and window. If so,
         Returns
         -------
         coef
-            The coefficients of`other` if it is a compatible instance,
-            of ABCPolyBase, otherwise `other`.
+            The coefficients of `other` if it is a compatible instance
+            of :class:`ABCPolyBase`, otherwise `other`.
```

Comment:
this looks incorrect, since `ABCPolyBase` isn't public.

Reply:
`ABCPolyBase` is both private and abstract. The fact that it is private is not really an issue for me as the method `_get_coefficients` is also private.  Since `ABCPolyBase` is abstract, perhaps it would be better to write `... if it is a compatible instance of a subclass of :class:'ABCPolyBase'`?

Reply:
Ah wait, this is in a docstring of private API, so it won't end up in the docs either way. In that case, it's okay to leave it as it is in this PR. It's only extra markup that isn't used, so I'd have chosen not to touch it. But it doesn't matter much either way.